preliz/distributions/logitnormal.py

```diff
@@ -149,6 +149,37 @@ def kurtosis(self):
         pdf = self.pdf(x_values)
         return np.trapz(((x_values - mean) / std) ** 4 * pdf, x_values) - 3
 
+    def mode(self):
```

Comment:
Move this to the optimization module and rename to something line find_mode_logitnormal